java/test/jmri/jmrix/dccpp/DCCppSensorManagerTest.java

```diff
@@ -41,11 +41,18 @@ public void testByAddress() {
         DCCppInterfaceScaffold xnis = new DCCppInterfaceScaffold(new DCCppCommandStation());
         Assert.assertNotNull("exists", xnis);
 
+        // create and register the manager object in a new instance manager
```

Comment:
I think you want to use JUnitUtil.resetInstanceManager() instead of creating your own.